library task/storage.py

```diff
@@ -3,7 +3,6 @@
 
 FILE_PATH = 'library.json'
 
-
```

Comment:
Not cool :(
Please fix before the day end or you are fired.
If you do, you are watered !!!

Reply:
You are fired